overlay/tun_openbsd.go

```diff
@@ -160,8 +170,8 @@ func (t *tun) addRoutes(logErrors bool) error {
 			// We don't allow route MTUs so only install routes with a via
 			continue
 		}
-
-		cmd := exec.Command("/sbin/route", "-n", "add", "-inet", r.Cidr.String(), t.cidr.Addr().String())
+		//todo is this right?
```

Comment:
is it okay to only use the zeroth IP for this on BSDs?

Reply:
I am fine to hand wave non darwin bsds for now just to get tests passing. It looked like we may be able to reuse a lot of the darwin netlink stuff there as well.